utils.py

```diff
@@ -457,32 +457,24 @@ def ann_in(name, lst):
     return [x for x in annotation_fields if not ann_in(x.name, ignore_list)]
 
 
-def pc_project(mt, pc_mt, pca_loadings_root='va.pca_loadings'):
+def pc_project(mt: hl.MatrixTable, pc_loadings: hl.Table):
     """
     Projects samples in `mt` on PCs computed in `pc_mt`
-    :param mt: MT containing the samples to project
-    :param pc_mt: MT containing the PC loadings for the variants
-    :param pca_loadings_root: Annotation root for the loadings. Can be either an Array[Double] or a Struct{ PC1: Double, PC2: Double, ...}
-    :return: MT with
-
-    pc_mt = pc_mt.annotate_variants_expr('va.pca.calldata = gs.callStats(g => v)')
+    :param MatrixTable mt: MT containing the samples to project (assumes data in `loadings` and `pca_af`)
```

Comment:
Is that paren meant for next line?

Also, I usually pass these as `str` params to get the little extra flexibility. But I suppose both ways are ok.

Reply:
Yes, it is. Will fix. And yeah, I thought about passing as strings, but it's nuking annotations in `mt` so its sorta dangerous to let them be called anything. But I guess that's the user's problem.